### DISCUSSION TITLE
Is there any way to annotate secrets created by operator

### DISCUSSION THREAD

**Discussion by bahaeddinoz** [strimzi/strimzi-kafka-operator#8838] [Q&A] [answered]
Hello,
i couldn't find on the notes but i want to use cluster certificates for my applications on another namespace which requires me to provide secret on another namespace. I wanted to use a app to sync secret to other namespace but this requires me to annotate created secrets. I couldn't find something on notes about how. Is there any way to do it?

**Comment by scholzj** [comment]
You can do that using the `template` sections -> for the cluster CA public key secret and for the User secrets.

**Reply by bahaeddinoz** [reply]
Hi @scholzj. Thank you a lot 🙏 . you are like a superman! How can i specify it on template? Only found this : https://strimzi.io/docs/operators/latest/configuring.html#type-MetadataTemplate-reference which only pods are getting the annotations.

**Reply by scholzj** [reply, marked as answer]
Yeah, basically. If you check the resource spec, it is used here for the Kafka part: https://strimzi.io/docs/operators/latest/configuring.html#type-KafkaClusterTemplate-reference ... so it will be somehting like `.spec.kafka.template.clusterCaCert.metadata.annotations. For the users, it will be something like `.spec.template.secret.metadata.annotations` in `KafkaUser`: https://strimzi.io/docs/operators/latest/configuring.html#type-KafkaUserTemplate-reference

**Reply by bahaeddinoz** [reply]
ohh i see. It worked! Thanks a lot again @scholzj 🙏 🙏
